apps/scalable/keepassx.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg height="48" width="48" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><linearGradient id="a" gradientUnits="userSpaceOnUse" x1="510.619519" x2="510.901121" y1="550.879889" y2="503.338722"><stop offset="0" stop-color="#14a102"/><stop offset="1" stop-color="#80e950"/></linearGradient><g stroke-width="1.317422" transform="translate(-486.76034 -502.725835)"><path d="m533.00453 512.86128c-.0147-1.12294-.87882-2.05059-1.99683-2.15317-9.29112-.86906-16.87346-5.83441-19.37317-7.65064-.52241-.38083-1.23035-.38083-1.75277 0-2.49488 1.81623-10.07717 6.78158-19.36828 7.65064-1.11806.10258-1.97735 1.03023-1.99688 2.15317-.1123 7.32841 1.06435 32.20882 21.68252 37.74545.36617.0977.75188.0977 1.11806 0 20.61816-5.5317 21.79476-30.41709 21.68735-37.74545z" fill="url(#a)"/><path d="m23.970703 5.8300781c-.2364 0-.473078.0729581-.673828.2207031-1.91746 1.4092601-7.744026 5.2631798-14.8847656 5.9374998-.85929.0796-1.5201463.798602-1.5351563 1.669922-.0863 5.6863.8178425 24.993043 16.6640629 29.289063.28143.0758.577945.0758.859375 0 15.84622-4.29219 16.750528-23.602803 16.667968-29.289063-.0113-.87132-.673953-1.590332-1.533203-1.669922-7.14075-.67432-12.969455-4.5282397-14.890625-5.9374998-.20075-.147745-.437428-.2207031-.673828-.2207031zm2.986328 10.1855469c2.77982 0 5.027344 2.247544 5.027344 5.027344 0 2.77979-2.247524 5.027343-5.027344 5.027343-.65058 0-1.241117-.119454-1.773437-.296874l-2.66211 2.662109h-1.77539v1.775391h-1.773438v1.773437h-2.957031v-2.957031l6.210937-6.210938c-.17742-.59145-.296874-1.181987-.296874-1.773437 0-2.77979 2.247553-5.027344 5.027343-5.027344zm1.478516 1.773437c-1.00547 0-1.773438.769911-1.773438 1.775391s.767968 1.773438 1.773438 1.773438c1.00548 0 1.775391-.767958 1.775391-1.773438s-.769911-1.775391-1.775391-1.775391z" fill="#fff" fill-opacity=".85098" transform="translate(486.76034 502.72583)"/></g></svg>
```

Comment:
This can probably just be a symlink to `apps/scalable/keepass.svg`.

Reply:
Symlinked `keepassx.svg` to `keepass.svg` as requested